superset/connectors/sqla/views.py

```diff
@@ -411,6 +409,36 @@ def _delete(self, pk):
         db.session.commit()
         self.update_redirect()
 
+    def get_deployment_names(self):
+        from sqlalchemy import create_engine
+        deployments = []
+        presto_engine = create_engine('presto://localhost:3894')
+        presto_connection = presto_engine.connect()
+        query = 'SHOW CATALOGS'
+        try:
+            results = presto_connection.execute(query)
+            for row in results:
+                deployment = row['Catalog']
+                if '_mysql' in deployment:
+                    deployment = deployment.split('_mysql')[0]
+                deployments.append(deployment)
+        except:
+            deployments = [
```

Comment:
This is a list of dictionaries, not a list of strings. The above try statement would not be compatible

Reply:
What kind of UI behavior do we want for alert field. Is it a dropdown as well? If so what are the options?